### ISSUE TITLE
[MEM-64] [BUG] Fonts don't scale

### ISSUE THREAD

**Issue by dan980** [gkasdorf/memmy#162]
**Describe the bug**
There is an accessibility issue in the app (version 72). The font size for headers, body and comments is fixed and does not follow system settings. Only the font for the buttons does scale. Also the accessibility option does not have any effect (specific text size per app).

**To Reproduce**

1. go to "settings" > Display & Brightness, then select Text Size.
2. drag the slider to select the text size that you want.
3. open the app

as an alternative:

1. go to "settings > Accessibility > Per-App Settings
2. add app
3. select "Memmy app"
4. select "larger text"
5. drag the slider to select the text size that you want for this app.
6. open the app

**Expected behavior**
All fonts should scale according to your settings.

**Screenshots**
Here screenshots of settings and home, using a big font size:

![](https://uploads.linear.app/7655b776-38c4-4273-b069-78045fe3cd9c/7e5d0ee7-7318-40c9-b7ee-44b18d99a307/ac3c606c-3f92-4b5f-92a2-0c41a2722ab1)

![](https://uploads.linear.app/7655b776-38c4-4273-b069-78045fe3cd9c/153ae12c-e9d3-4c8e-8f85-c3a9a5543381/6877b9b6-b42d-4707-b9f0-a0e16e4e5174)

**Smartphone (please complete the following information):**

* Device: iPhone 12 pro max
* OS: iOS 16.5.1
* Version 72

**Additional context**
This problem hinders usage of the app for the visually impaired users.

<sub>From [SyncLinear.com](https://synclinear.com) | [MEM-64](https://linear.app/memmyapp/issue/MEM-64/[bug]-fonts-dont-scale)</sub>
